### ISSUE TITLE
[FluxStore Pro] - adjusting home page layout with status bar on IOS and Android

### ISSUE THREAD

**Issue by Rileyjrjohns** [inspireui/support#2500]
> - Thank you for reaching please follow the format below to create a new ticket, please explain as more detail as possible and allow 24 - 48 hours to receive an answer to your ticket.
> - Try to explain your all problems in one ticket, instead of creating multi tickets.
> - After creating an issue, please take the screenshot of http://codecanyon.net/download - to let us know you have purchased our product, or quick note as a comment that you have created this ticket, example: https://tppr.me/DJA8L
> - IMPORTANT: We will close the ticket if you could not prove as our customer.

---------

1. My issue:

I've an issue with Home Page layout. When i try to set Home page Layout 6 or 1 for exemple, the status bar is still white. How could i do for have the same render as preview demo ?
I copied the code to config_en.json but i don't find the setting for this..
I got the issue on IOS but also on Android.

Thanks for your help !

2. Screenshots issues (required):

![Capture d’écran 2020-03-31 à 13 13 51](https://user-images.githubusercontent.com/58285390/78020498-dcab9900-7351-11ea-843a-afdf8c04d277.png)
![Capture d’écran 2020-03-31 à 13 17 01](https://user-images.githubusercontent.com/58285390/78020535-ef25d280-7351-11ea-80c5-5a45686a21ed.png)

3. Proof of my purchased (required):

![Capture d’écran 2020-03-30 à 17 10 28](https://user-images.githubusercontent.com/58285390/77929406-eaa5df00-72a9-11ea-8028-7a1e18b5c27c.png)

3. Config files (required)

![Capture d’écran 2020-03-30 à 17 10 45](https://user-images.githubusercontent.com/58285390/77929425-f1345680-72a9-11ea-9f0f-f35adfb4fe53.png)

**Comment by philipvu30** [comment]
Hi @Rileyjrjohns ,

Please add this line 
await SystemChrome.setEnabledSystemUIOverlays([SystemUiOverlay.bottom]);
to this file in this path: lib/main.dart

![image](https://user-images.githubusercontent.com/54442328/78037054-397c7300-7395-11ea-8ba5-afef37a668eb.png)

**Comment by Rileyjrjohns** [comment]
Hello, @philipvu30,

it only hide Status bar but doesn't extend banner as you can see here 
![Capture d’écran 2020-03-31 à 16 31 30](https://user-images.githubusercontent.com/58285390/78038400-18a02780-736d-11ea-93ec-aa858c21c05f.png)

thanks for your help 👍

**Comment by philipvu30** [comment]
I don't know about IOS but on the Android emulator, the status bar is hidden with the code above.
Maybe you can change it to **await SystemChrome.setEnabledSystemUIOverlays([]);**.

**Comment by minhcasi** [comment]
@philipvu30 - this issue could be fixed on the latest code, pls double check this example

**Comment by Rileyjrjohns** [comment]
I hardly try to find a way without any issue actually :/

**Comment by minhcasi** [comment]
Could you try to fix this issue by edit home/home.dart file: 

before: https://tppr.me/FOIBU
after: https://tppr.me/FOIBU

**Comment by Rileyjrjohns** [comment]
> Could you try to fix this issue by edit home/home.dart file:
> 
> before: https://tppr.me/FOIBU
> after: https://tppr.me/FOIBU
> 
> > <img alt="" width="48" src="https://camo.githubusercontent.com/9bfbde510e674b28784a90f17f04987914740ec1/68747470733a2f2f747070722e73332e616d617a6f6e6177732e636f6d2f75706c6f6164732f34383831623364313466356134633239663138613937363437663933326461312e706e67"><img alt="" height="14" src="/favicon-32x32.png"> tppr.me**[Public - Android Studio_2020-04-01 15-40-54](https://tppr.me/FOIBU)**

Excellent ;) You might add this to your docs in home layout for the themes that need it (1, 6, 20 ... etc). Thanks a lot
